package.json

```diff
@@ -29,34 +29,34 @@
     "version": "yarn run build && git add -A"
   },
   "devDependencies": {
-    "@babel/cli": "7.8.4",
-    "@babel/core": "7.9.6",
-    "@babel/plugin-proposal-class-properties": "7.8.3",
-    "@babel/plugin-proposal-object-rest-spread": "7.9.6",
-    "@babel/plugin-transform-object-assign": "7.8.3",
-    "@babel/polyfill": "7.8.7",
-    "@babel/preset-env": "7.9.6",
-    "@babel/preset-react": "7.9.4",
+    "@babel/cli": "7.10.1",
+    "@babel/core": "7.10.2",
+    "@babel/plugin-proposal-class-properties": "7.10.1",
+    "@babel/plugin-proposal-object-rest-spread": "7.10.1",
+    "@babel/plugin-transform-object-assign": "7.10.1",
+    "@babel/polyfill": "7.10.1",
+    "@babel/preset-env": "7.10.2",
+    "@babel/preset-react": "7.10.1",
     "@rollup/plugin-commonjs": "11.1.0",
     "@rollup/plugin-node-resolve": "7.1.3",
     "@rollup/plugin-replace": "2.3.2",
     "@svgr/rollup": "5.4.0",
     "@svgr/webpack": "5.4.0",
-    "@testing-library/jest-dom": "5.8.0",
-    "@testing-library/react": "10.0.4",
+    "@testing-library/jest-dom": "5.9.0",
+    "@testing-library/react": "10.2.0",
     "@testing-library/react-hooks": "3.2.1",
     "@types/jest": "25.2.3",
     "@types/prop-types": "15.7.3",
     "@types/react": "^16.9.33",
     "@types/react-dom": "^16.9.6",
-    "@types/styled-components": "4.4.3",
-    "@typescript-eslint/eslint-plugin": "2.31.0",
-    "@typescript-eslint/parser": "2.31.0",
-    "@zendeskgarden/css-bedrock": "8.0.0",
+    "@types/styled-components": "5.1.0",
```

Comment:
@hzhu are we able to upgrade this while still using `styled-components@v4`? We may need to pin this until we can upgrade without styleguidist issues.

Reply:
>  are we able to upgrade this while still using styled-components@v4?

I thought so — mostly because the build passed successfully and I didn't encounter any issues with `yarn tsc`, or any other local `yarn` commands.

With the upgrade from TypeScript `v3.8.3` to `3.9.3`, we are seeing this [TS error](https://app.circleci.com/pipelines/github/zendeskgarden/react-components/837/workflows/5c4b2b9c-252c-4a2e-b41c-2762a1f6845f/jobs/1465). Upgrading `@types/styled-components` to `v5.1.0` [resolves](https://github.com/microsoft/TypeScript/issues/37597#issuecomment-628149946) the error.

Alternatively, we could keep TypeScript at `v3.8.3` and not upgrade to `v3.9.3`. @austin94, what do you think is the best path forward?

Reply:
Even though our usage works with `@types/styled-components@v5` we shouldn't use it as TS checking could pass, but not match the API in our version.

[From the open issue](https://github.com/microsoft/TypeScript/issues/37597) we could copy the type definitions locally, but I would rather us just hold off on upgrading Typescript until then to avoid fragmenting from the official types.

Reply:
> rather us just hold off on upgrading Typescript until then to avoid fragmenting from the official types

I agree, this sounds good 👍 .